apis/env.d.ts

```diff
@@ -64,23 +64,50 @@ export namespace env {
       binding?: Binding,
       [key: string]: any,
     },
-    multitenancy: boolean | { kind: string, jobs: {
+    multitenancy?: boolean | { kind: string, jobs: {
       clusterSize: number,
       workerSize: number,
       t0: string,
       [key: string]: any,
     },},
-    toggles: boolean,
-    extensibility: boolean | {
+    toggles?: boolean,
+    extensibility?: boolean | {
       model: string[],
       tenantCheckInterval: number,
       [key: string]: any,
     },
-    messaging: {
+    messaging?: {
       kind: 'file-based-messaging' | 'redis-messaging' | 'local-messaging' | 'enterprise-messaging' | 'enterprise-messaging-shared' | string,
       format: 'cloudevents' | string,
       [key: string]: any,
     },
+    'cds.xt.SaasProvisioningService'?: {
+      model: string,
+      kind: string,
+      alwaysUpgradeModel?: boolean,
+      [key: string]: any,
+    },
+
+    'cds.xt.SmsProvisioningService'?: {
+      model: string,
+      kind: string,
+      [key: string]: any,
+    },
+    'cds.xt.ExtensibilityService'?: {
+      model: string,
+      'namespace-blocklist': string[],
+      'extension-allowlist': { for: string[], 'new-entities'?: number }[],
+      [key: string]: any,
+    },
+    'cds.xt.ModelProviderService'?: {
```

Comment:
I'm seeing a `kind` key [in the example](https://cap.cloud.sap/docs/guides/multitenancy/mtxs#combined-with-convenience-flags), or is that an unrelated section?